tests/test_datetime_import.rs

```diff
@@ -1,6 +1,6 @@
 #![cfg(not(Py_LIMITED_API))]
 
-use pyo3::{types::PyDate, Python};
+use pyo3::{prelude::PyAnyMethods, types::PyDate, Python};
```

Comment:
```suggestion
use pyo3::{prelude::*, types::PyDate};
```

Slightly related thought: I've been thinking we might want to make these traits public at some path other than the prelude, just for the few occasions when users might want to directly name them (?).

One option would be to make their existing paths public, e.g. `pyo3::types::any::PyAnyMethods`. We'd just need to be careful we didn't accidentally make other APIs public at the same time.

Reply:
`pyo3::types` I think makes a lot of sense. That module is already public, and we can control easily what gets exposed 👍